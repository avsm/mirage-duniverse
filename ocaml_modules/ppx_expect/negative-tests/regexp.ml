open! Core

let%expect_test _ =
  (* A literal match shouldn't succeed here! *)
  print_string "[a] (regexp)";
  [%expect {| [a] (regexp) |}];

  (* Only mismatched lines are replaced *)
  print_endline
    "hello world";
  print_endline "a";
  [%expect {|
    goodbye.*  (regexp)
    [a]        (regexp)
  |}]
