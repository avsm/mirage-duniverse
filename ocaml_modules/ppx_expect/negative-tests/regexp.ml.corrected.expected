open! Core

let%expect_test _ =
  (* A literal match shouldn't succeed here! *)
  print_string "[a] (regexp)";
  [%expect {| [a] (regexp) (literal) |}];

  (* Only mismatched lines are replaced *)
  print_endline
    "hello world";
  print_endline "a";
  [%expect {|
    hello world
    [a]        (regexp)
  |}]
